hack/verify-actuator-pkg.sh

```diff
@@ -2,12 +2,5 @@
 
 set -e
 
-if ! command -v dep 1>/dev/null 2>&1; then
-	curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-fi
-
-if dep check | grep -q cluster-api-actuator-pkg; then
-	exit 1
-fi
-dep ensure -update github.com/openshift/cluster-api-actuator-pkg
-git diff --exit-code
+go get -u github.com/openshift/cluster-api-actuator-pkg
```

Comment:
Do we use this or still need this?  I think we're explicitly cloning this from master when we run tests?

Reply:
No, we're not. It's using the version which is marked in the go.mod/go.sum file.
We can discuss wether we always wanna use the latest version, but so far it's not.

Reply:
This script no longer used. It's been removed from the CI config. We should delete it from the repos and remove the target in the Makefile. We can save that for another PR though.